.github/workflows/chain-endpoints.yml

```
@@ -1,10 +1,11 @@
 name: Chain endpoints
 on:
   schedule:
-    - cron:  '45 0/12 * * *'
+    - cron:  '50 0/12 * * *'
 
 jobs:
   types:
+    if: (github.event_name == 'schedule' && github.repository == 'polkadot-js/apps') || (github.event_name != 'schedule')
```

```suggestion
    if: (github.event_name == 'schedule' && github.repository == 'polkadot-js/apps')
```

Would just keep it at schedule. Can be run manually (locally) if need-be.